src/woeusb

```diff
@@ -2004,6 +2009,19 @@ util_check_function_parameters_quantity(){
 	return 0
 }; declare -fr util_check_function_parameters_quantity
 
+##Check every file in source directory for files bigger than max fat32 file size (~4GB)
+check_for_big_files(){
+	local source_fs_mountpoint="${1}"
+
+	for i in $(find ${source_fs_mountpoint}/ -type f); do
+		if (( $(du $i | cut -f -1) > 4194303 )); then
```

Comment:
* `stat(1)` should be more suitable than `du(1)` to determine file size
* Why `4194303`?  Document the unit conversion in the above comments